src/GitHub.InlineReviews/Models/PullRequestSessionFile.cs

```diff
@@ -35,6 +35,27 @@ public PullRequestSessionFile(string relativePath)
             RelativePath = relativePath;
         }
 
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        bool disposed = false;
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposed)
+            {
+                disposed = true;
+
+                if (disposing)
+                {
+                    linesChanged.Dispose();
```

Comment:
A `PullRequestSessionFile` can be a shared instance between multiple clients, disposing of `linesChanged` here would unsubscribe all clients. IMO this class shouldn't be disposable, or at least the `Dispose` method shouldn't dispose `linesChanged` as this will break things.